features/step_definitions/networking.rb

```diff
@@ -733,7 +733,7 @@
   ensure_admin_tagged
   node = node(node_name)
   host = node.host
-  @result = host.exec_admin("/sbin/ip link delete #{bridge_name}")
+  @result = host.exec_admin("if ip addr show  #{bridge_name};then ip link delete #{bridge_name};else echo #{bridge_name} does not exist on this node;fi")
```

Comment:
@anuragthehatter make make it kind of hard to read...would it be possible to pull bash script out, assign it a variable and then call that instead?

Reply:
Thanks Peter, Fixed. Not seeing any premature exists errors even if inf doesn't exist on a node https://privatebin-it-iso.int.open.paas.redhat.com/?a7333e1ec1d3b598#DfdLk89I4fMQ3rx6CKcd36kgC5Csj7REN1LA1JNKFoA=